### PR TITLE
Signup service

### PR DESCRIPTION
**Pull request by tomohung**

### CODE REVIEW

File: app/models/video.rb [hunk 1]
```diff
@@ -13,4 +13,8 @@ def self.search_by_title(search_term)
     where('title LIKE ?', "%#{search_term}%").order(created_at: :desc)
   end
  
+  def rating
+    reviews.average(:rating).round(1) if reviews.average(:rating)
```
Comment (brandoncc) [inline app/models/video.rb:17]: Nice job pushing the work to the database. You are the first person I have seen use this here!

Comment (brandoncc) [inline app/models/video.rb:17]: I mean the `average` method, by the way.

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -42,21 +42,18 @@
...
-      end
-
-      it 'should create user' do
-        expect(User.count).to eq(1)  
-      end
 
-      it 'should logged_in if create user' do
-        expect(session[:user_id]).to eq(User.first.id)
-      end
+      # it 'should logged_in if create user' do
```
Comment (brandoncc) [inline spec/controllers/users_controller_spec.rb]: Make sure you cleanup any unused code which will just clutter your codebase.

File: app/models/video.rb (hunk 1 above)
Comment (tomohung) [inline app/models/video.rb:17]: I just copy it from Kevin's code :+1:  haha!

**Pull request merged**